src/Manager.js

```diff
@@ -218,7 +243,8 @@ class GiveawaysManager extends EventEmitter {
                 embedColor: options.embedColor,
                 embedColorEnd: options.embedColorEnd,
                 extraData: options.extraData,
-                lastChance: options.lastChance
+                lastChance: options.lastChance,
+                pauseOptions: typeof options.pauseOptions === 'object' ? options.pauseOptions : defaultPauseOptions 
```

Comment:
```suggestion
                pauseOptions: typeof options.pauseOptions === 'object' ? options.pauseOptions : null
```
Saves db space = look at getter change